sentry-spring/src/main/java/io/sentry/spring/tracing/SentrySpanAdvice.java

```diff
@@ -26,6 +26,7 @@ public SentrySpanAdvice(final @NotNull IHub hub) {
     this.hub = Objects.requireNonNull(hub, "hub is required");
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
spring-boot deprecated StringUtils.isEmpty() but I don't want to change it because not everybody else will be using latest spring boot